### PR TITLE
HTBHF-1972 Updated to latest version of selenium and enabled config switch for headless mode

### PR DESCRIPTION
**Pull request by YetAnotherMatt**
The latest version of selenium seems to work with headless chrome, so the switch I've added here isn't strictly necessary. Seems nicer than having to edit code to change between modes though.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,18 +8,16 @@ cache:
   directories:
     - bin/
 
+dist: bionic
```
Comment (mikeSmith08) [inline .travis.yml]: What does this do?

File: src/test/common/config.js [hunk 2]
```diff
@@ -7,11 +7,13 @@ const SESSION_DETAILS_BASE_URL = process.env.SESSION_DETAILS_BASE_URL || `http:/
 const SESSION_DETAILS_PATH = '/session-details/confirmation-code'
 
 module.exports.PORT = process.env.PORT
-module.exports.TEST_BROWSER = process.env.TEST_BROWSER || 'chrome'
+module.exports.TEST_BROWSER = 'chrome'
```
Comment (YetAnotherMatt) [inline src/test/common/config.js]: I'll revert this once all branches have inherited this and we can remove the env setting in travis

Comment (mikeSmith08) [inline src/test/common/config.js]: Is there any point in keeping this in? Same with the value in `sample.env` too.

File: .travis.yml (hunk 1 above)
Comment (YetAnotherMatt) [reply .travis.yml]: Changes the version of ubuntu used by travis. Bionic is the most recent LTS. It was needed for xvfb, but we don't need xvfb now...

**Pull request merged**